service/history/queues/queue_base.go

```diff
@@ -328,16 +333,23 @@ func (p *queueBase) checkpoint() {
 			InclusiveMinTaskKey: persistenceMinTaskKey,
 			ExclusiveMaxTaskKey: persistenceMaxTaskKey,
 		}); err != nil {
+			p.logger.Error("Error range completing queue task", tag.Error(err))
 			return
 		}
 
 		p.inclusiveLowWatermark = newInclusiveLowWatermark
 	}
 
+	p.metricsHandler.Counter(AckLevelUpdateCounter).Record(1)
+	// p.metricsHandler.Histogram(PendingTasksCounter, metrics.Dimensionless).Record(int64(p.monitor.GetPendingTasksCount()))
```

Comment:
any replacement for this metric?

Reply:
Will uncomment this after queue monitor code is landed.